test/e2e/common/storage/downwardapi_volume.go

```diff
@@ -339,7 +339,7 @@ func downwardAPIVolumeBaseContainers(name, filePath string) []v1.Container {
 				},
 				Limits: v1.ResourceList{
 					v1.ResourceCPU:    resource.MustParse("1250m"),
-					v1.ResourceMemory: resource.MustParse("64Mi"),
+					v1.ResourceMemory: resource.MustParse("128Mi"),
```

Comment:
is this value checked later in the tests?

Reply:
Yes

```log
   <*errors.errorString | 0xc000f16a30>: 
      expected "67108864\n" in container output: Expected
          <string>: content of file "/etc/podinfo/memory_limit": 134217728
          
      to contain substring
          <string>: 67108864
```